src/frontend/planner_test/tests/testdata/insert.yaml

```diff
@@ -41,17 +74,18 @@
     BatchExchange { order: [], dist: Single }
     └─BatchInsert { table: t }
       └─BatchValues { rows: [['2020-01-01 01:02:03':Varchar::Timestamp::Time], ['03:04:05':Varchar::Time]] }
-- name: a `VALUES` without insert context may be invalid on its own (compare with
+- name:
+    a `VALUES` without insert context may be invalid on its own (compare with
     above)
   sql: |
     create table t (v1 time);
     values (timestamp '2020-01-01 01:02:03'), (time '03:04:05');
-  binder_error: 'Bind error: types Timestamp and Time cannot be matched'
+  binder_error: "Bind error: types Timestamp and Time cannot be matched"
```

Comment:
Do you generate this file with `./risedev do-apply-planner-test`? 🤔

Reply:
No. I wrote this manually and on a different occasion I wrote a python scripts that writes tests for me. Why? Is there an issue with the quoting?

Reply:
Just tried the tool. Will upload the tool-version to stay consistent with the "code style" of the tests.

Reply:
Not a big deal. I guess it will be overridden with some next PRs. 🥰

Reply:
Still. Good catch 👍